app/helpers/url_helper.rb

```diff
@@ -24,15 +24,10 @@ def govuk_domain_suffix(environment, on_aws:)
     "#{environment}.publishing.service.gov.uk"
   end
 
-  def jenkins_deploy_app_url(application, release_tag, environment)
+  def jenkins_deploy_url(application, release_tag, environment)
     suffix = govuk_domain_suffix(environment, on_aws: application.on_aws?)
+    job_name = "Deploy_Puppet" if application.shortname == "puppet"
```

Comment:
Think this needs to be a ternary:

```suggestion
    job_name = application.shortname == "puppet" ? "Deploy_Puppet" : "Deploy_App"
```

Reply:
Sure, I've changed it now.